pubtools/_quay/push_docker.py

```diff
@@ -499,6 +503,9 @@ def remove_old_signatures(
         outdated_signatures = []
 
         for image_data, manifest in backup_tags.items():
+            # do not remove signatures for just pushed content in the case of a repush
+            if image_data in rollback_tags:
```

Comment:
If you push exactly same push twice. Then in the first push there will be tags lets say v1,v1.1,latest in the rollback_tags as those are newly added. In the second push, tags v1,v1.1,latest will be marked as backup_tags as those are already there and as a rollback_tags will be set the same tags because you want to add them in the push.

Reply:
> rollback_tags will be set the same tags because you want to add them in the push.

In `generate_backup_mapping`, rollback_tags is added when repo/digest/tag does not exist. How are those tags added to rollback_tags in the second push?